src/breakpoints/index.ts

```diff
@@ -130,13 +111,32 @@ export namespace Breakpoints {
       Signal.clearData(this);
     }
 
-    private _selectedType: SessionTypes;
+    setBreakpoints(id: string, breakpoints: IBreakpoint[]) {
+      this._breakpoints.set(id, breakpoints);
+      this.changed.emit(breakpoints);
+    }
+
+    getBreakpoints(id: string): IBreakpoint[] {
+      return this._breakpoints.get(id) || [];
+    }
+
+    restoreBreakpoints(breakpoints: Map<string, IBreakpoint[]>) {
+      this._breakpoints = breakpoints;
```

Comment:
Is there a reason for not using the `breakpoints` setter that will emit the `changed` signal?

Reply:
Never mind that comment, the setter is removed in this change.